src/segment-loader.js

```diff
@@ -2968,6 +2969,8 @@ export default class SegmentLoader extends videojs.EventTarget {
     // used for testing
     this.trigger('appended');
 
+    this.mediaAppends++;
```

Comment:
This will show extra appends for:
* segments with no data to be appended
* sync requests that did not get appended

It will also under-report total number of appends, since we potentially append multiple times per segment.

Do we want to consider those cases and think about naming as "segmentAppends" since that may denote full segments?

Reply:
sync requests that do not get appended will return much earlier in the function. We could wrap `mediaAppends` increment in `segmentInfo.hasAppendedData`?

Reply:
the other problem with calling it segment appends is that we now append parts too.

Reply:
👍 on sync requests

On wrapping and part appends, what is our use-case for this data? Do we want to know actual appends, or are we more concerned about requests?

Reply:
I think we want to know actual appends, we already have stats for requests.